File: src/main/java/frc/robot/commands/Climber_Commands/ManualClimber.java
package frc.robot.commands.Climber_Commands;

import edu.wpi.first.wpilibj2.command.Command;
import frc.robot.subsystems.Climber;

public class ManualClimber extends Command {

    private final Climber climber;
    private final double climberVel;

    // Use addRequirements() here to declare subsystem dependencies.
    public ManualClimber(Climber climber, double climberVel) {
        this.climber = climber;
        this.climberVel = climberVel;
        addRequirements(climber);
    }

    @Override
    public void initialize() {
        System.out.println("INFO: ManualClimber.initialize");
    }

    @Override
    public void execute() {
        climber.setManual(climberVel);
    }

    @Override
    public void end(boolean interrupted) {
        climber.stop();
    }

    @Override
    public boolean isFinished() {
        return false;
    }
}

File: src/main/java/frc/robot/commands/Climber_Commands/ManualClimber.kt
package frc.robot.commands.Climber_Commands

import edu.wpi.first.wpilibj2.command.CommandBase
import frc.robot.subsystems.Climber

class ManualClimber(private val climber: Climber, private val climberVel: Double) : CommandBase() {

    // Use addRequirements() here to declare subsystem dependencies.
    init { addRequirements(climber) }

    // Called when the command is initially scheduled.
    override fun initialize() {}

    // Called every time the scheduler runs while the command is scheduled.
    override fun execute() { climber.setManual(climberVel) }

    override fun end(interrupted: Boolean) { climber.stop() }

    override fun isFinished(): Boolean { return false }

}